controllers/submission.go

```diff
@@ -60,7 +60,7 @@ func PostSubmission(ctx context.Context, w http.ResponseWriter, r *http.Request)
 	}
 
 	taskKey, err := datastore.DecodeKey(mux.Vars(r)["taskKey"])
-
+	// TODO: When more task kinds are added, see controllers/result.go line 156.
```

Comment:
This is not a `TODO`, it is a `NOTE`. Also, I'd rather not refer to a line number as they are *very* fragile. I even have the feeling this comment was already outdated at the time of the commit, as `controllers/result.go:156` reads:

```go
return http.StatusBadRequest, errors.New("Unknown submission kind.")
```

Please refer to to `controllers.CreateFinalResult`.